src/main/kotlin/no/nav/familie/ef/iverksett/økonomi/simulering/SimuleringUtil.kt

```diff
@@ -122,3 +123,26 @@ fun BeriketSimuleringsresultat.harFeilutbetaling(): Boolean {
     return this.oppsummering.feilutbetaling > ZERO
 }
 
+
+fun Simuleringsoppsummering.hentSammenhengendePerioderMedFeilutbetaling(): List<Periode> {
+    val perioderMedFeilutbetaling =
+            perioder.sortedBy { it.fom }.filter { it.feilutbetaling > BigDecimal(0) }.map {
```

Comment:
Har ikke så mye å si, men kan bruke BigDecimal.ZERO :P

Reply:
(burde kanskje 1-2 tester for PR'en? )

Reply:
Jeg copy-pasta rått fra ef-sak 😂

Reply:
Fiksa testen som feilet til å teste dette